app/models/manageiq/providers/container_manager.rb

```diff
@@ -46,9 +47,17 @@ class ContainerManager < BaseManager
     has_many :all_container_images, :foreign_key => :ems_id, :dependent => :destroy, :class_name => "ContainerImage"
     has_many :all_container_nodes, :foreign_key => :ems_id, :dependent => :destroy, :class_name => "ContainerNode"
 
+    has_one :virtualization_manager,
+            :foreign_key => :parent_ems_id,
+            :class_name  => "ManageIQ::Providers::Kubevirt::InfraManager",
+            :autosave    => true,
+            :dependent   => :destroy
```

Comment:
@moolitayer should this be in a provider repo like monitoring manager?
https://github.com/manageiq/manageiq-providers-openshift/blob/master/app/models/manageiq/providers/openshift/container_manager.rb#L19-19

Reply:
@zeari due to the inverse reference from https://github.com/ManageIQ/manageiq-providers-kubevirt/pull/6/files#diff-ff17b05a7c525eec833f0ddb0134a0d9R30  - it should reference a common class iiuc.